filebeat/module/kafka/log/manifest.yml

```diff
@@ -0,0 +1,14 @@
+module_version: 1.0
+
+kafka.home: /opt/kafka[_-\.0-9]*
```

Comment:
I think this could be made also a `var`, which can be referred from the `paths` array. This way, the user can only set that variable. Unless I missremember, the variables can refer to variables defined before them using the syntax `{{.home}}`.